python/items.json

```
@@ -0,0 +1 @@
+{"items": [{"name": "jacket", "category": "fashion", "image_filename": "9b4ea9d9ed1dfd8348e34a0520b9b979c3f4c8ed3635416ba2ea000cfb6b1046.jpg"}]}
```

[nit] 改行するとこのマークが消えるので試してみてください。
```suggestion
{"items": [{"name": "jacket", "category": "fashion", "image_filename": "9b4ea9d9ed1dfd8348e34a0520b9b979c3f4c8ed3635416ba2ea000cfb6b1046.jpg"}]}

```